### DISCUSSION TITLE
[@types/three] CCDIKSolver's IKS interface is wrong

### DISCUSSION THREAD

**Discussion by joaogardenberg** [DefinitelyTyped/DefinitelyTyped#66428] [Issues with a @types package]
As described in their [documentation](https://threejs.org/docs/#examples/en/animations/CCDIKSolver):

- `iteration` — (optional) Iteration number of calculation. Smaller is faster but less precise. Default is 1.
- `minAngle` — (optional) Minimum rotation angle in a step. Default is undefined.
- `maxAngle` — (optional) Maximum rotation angle in a step. Default is undefined.
- `links.enabled` — (optional) Default is true.

Those 4 properties are actually optional, while here they're required:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/4cb97e723e3d8ad78f354b920a2843903b2e6b7f/types/three/examples/jsm/animation/CCDIKSolver.d.ts#L4-L17

[a comment by an automated account is not shown]

**Comment by Methuselah96** [comment]
Apologies for the delayed response! This should be resolved by https://github.com/three-types/three-ts-types/pull/588 and released with `@types/three@0.156.0`. In the future if you create an issue at https://github.com/three-types/three-ts-types you're more likely to get a timely response/fix.
